cypress/support/config/settings.js

```diff
@@ -836,16 +836,16 @@ module.exports = () => ({
       featureIndexPage: {
         environments: {
           live: {
-            paths: [],
-            enabled: false,
+            paths: ['/arabic/magazine-54029177'],
+            enabled: true,
```

Comment:
We can't enable the live paths until they are also live in Simorgh, as this will cause tests to fail (they will be running against PAL pages instead of Simorgh pages)